src/app/wishlists/reducer.ts

```diff
@@ -78,36 +87,38 @@ export function saveWishListToIndexedDB() {
 export function loadWishListAndInfoFromIndexedDB(): ThunkResult<Promise<void>> {
   return async (dispatch, getState) => {
     if (!getState().wishLists.loaded) {
-      const wishListAndInfo = await get<WishListsState['wishListAndInfo']>('wishlist');
+      const wishListState = await get<WishListsState>('wishlist');
```

Comment:
You're going to need to deal with the fact that this was once saved in a different format, and handle old data.